.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,56 @@
+---
+name: Bug report
+about: Create a report to help us fix or improve an issue
+---
+
+**This is a bug report for the `@clr` Angular or UI versions of the design system.
+For the web-component implementation of Clarity (`@cds`), visit [vmware-clarity/core](https://github.com/vmware-clarity/core).**
+
+## Describe the bug
+
+_A clear and concise description of what the bug is, and the conditions when it occurs._
+
+## How to reproduce
+
+_Please provide a link to a reproduction scenario using one of the [Clarity Stackblitz templates](https://stackblitz.com/@clr-team). Reports without a clear reproduction may not be prioritized until one is provided._
+
+Steps to reproduce the behavior:
+
+1.  Go to '...'
+2.  Click on '....'
+3.  Scroll down to '....'
+4.  See error
+
+## Expected behavior
+
+_A clear and concise description of what you expected to happen._
+
+## Versions
+
+**Clarity version:**
+
+- [ ] v5.x
```

Comment:
I believe we need to remove this one cause it is out of support @ashleyryan

Reply:
Can do! I removed 3 and 4 for that reason, but figured 5 was still newly enough out of support it might make sense to keep it on here. `Other` covers it though.